frontend/javascripts/oxalis/view/context_menu.js

```diff
@@ -734,23 +734,31 @@ function ContextMenu(props: Props) {
           hideContextMenu();
         }}
       />
-      <div
-        style={{
-          position: "absolute",
-          left: contextMenuPosition[0],
-          top: contextMenuPosition[1],
-        }}
-        className="node-context-menu"
-        tabIndex={-1}
-        ref={inputRef}
-      >
-        <Shortcut supportInputElements keys="escape" onTrigger={hideContextMenu} />
-        {clickedNodeId != null
-          ? NodeContextMenuOptions({ ...props, clickedNodeId })
-          : NoNodeContextMenuOptions({ activeTool, segmentIdAtPosition, ...props })}
-
-        <Divider className="hide-if-first hide-if-last" style={{ margin: "4px 0px" }} />
-        {infoRows}
+      {/* This div serves as a "prison" for the sticky context menu. The above div
+      cannot be used since the context menu would then be closed on every click.
+      Since both divs are absolutely positioned and cover the whole page,
+      avoid blocking click events by temporarily disabling them for this "prison" div. */}
+      <div className="node-context-menu-overlay" style={{ pointerEvents: "none" }}>
+        <div
+          style={{
+            position: "sticky",
```

Comment:
Using Sticky is a very very nice idea 🎉  🚀

Reply:
:exploding_head: :+1: Didn't know that `sticky` can be used for this scenario. Grand stuff!